kwalitee/cli.py

```diff
@@ -96,24 +96,6 @@ def generate_json(ctx, output=None):
         print(json_str)
 
 
-@repo.command('ocrd-tool')
-@click.option('-o', '--output', help="Output file. Omit to print to STDOUT")
-@pass_ctx
-def generate_tool_json(ctx, output=None):
-    '''
-    Return one big list of ocrd tools
-    '''
-    ret = {}
-    _check_cloned(ctx)
-    for repo in ctx.repos:
-        ret = {**ret, **repo.get_ocrd_tools()}
-    json_str = json.dumps(ret, indent=4, sort_keys=True)
-    if output:
-        Path(output).write_text(json_str)
-    else:
-        print(json_str)
```

Comment:
I do use this for generating https://ocr-d.de/js/ocrd-all-tool.json though. I will gladly adapt it once the dust from the refactoring settles but not drop it entirely.

Reply:
Alrighty. As pointed out the underlying code has been dumped some time ago, but I will restore it.